Sources/MongoSwift/MongoClient.swift

```diff
@@ -312,8 +311,16 @@ public class MongoClient {
      * Starts a new `ClientSession` with the provided options and passes it to the provided closure.
      * The session is only valid within the body of the closure and will be ended after the body completes.
      *
-     * - Throws:
-     *   - `RuntimeError.compatibilityError` if the deployment does not support sessions.
+     * - Parameters:
+     *   - options: Options to use when creating the session.
+     *   - sessionBody: A closure which takes in a `ClientSession` and returns an `EventLoopFuture<T>`.
+     *
+     * - Returns:
+     *    An `EventLoopFuture<T>`, the return value of the user-provided closure.
```

Comment:
Could say "On success, the future contains ..." similar to the other docstrings.

Reply:
I opted to not do that here because we actually (from the user's perspective) directly return the return value of their closure (which happens to be a future because of how we've specified the type of `sessionBody`). regardless of whether that succeeds or fails we will directly return it